cmake/SetupCompiler.cmake

```diff
@@ -51,7 +51,7 @@ elseif(CMAKE_CXX_COMPILER_ID STREQUAL "Clang" OR CMAKE_CXX_COMPILER_ID STREQUAL
     set(xad_cxx_flags -Wall -Wshadow -Wconversion)
     if(CMAKE_CXX_COMPILER_ID STREQUAL "AppleClang")
         # otherwise we get clashes with complex headers and other things on MacOS
-        list(APPEND xad_cxx_flags -stdlib=libc++ -mmacosx-version-min=10.9)
+        list(APPEND xad_cxx_flags -stdlib=libc++ -mmacosx-version-min=10.9 -fbuiltin)
```

Comment:
This flag was accidentally pushed. This is fixed in the latest commit.

Reply:
The reason it originally made it there was because I was having issues with builtin functions, specifically __builitin_isinf and __builtin_isnan. I was experimenting with telling the compiler to not favour these builtin functions, but eventually found a workaround. Removing this flag has no effect on the build.